config/FeatureFlag.json

```diff
@@ -21,5 +21,6 @@
   "mixpanel": false,
   "business_profile": false,
   "generate_report": false,
+  "business_details": true,
```

Comment:
reason for this new feature flag?

Reply:
So we have now 4 new sections under the settings. So all those sections are now based upon the feature flag value